jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1552,4 +1552,12 @@ properties:
             title: Coronagraphic PSF Subtraction
             type: string
             fits_keyword: S_KLIP
+      bkglevel:
```

Comment:
I would suggest putting these 2 keywords into their own sub-section of the meta data, so that their corresponding FITS keywords also appear under a separate section heading in the FITS header, e.g.
`    skymatch:
        title: Skymatch background information
        type: object
        properties:
          bkglevel:
            title: Computed/matched background level
            type: float
            fits_keyword: BKGLEVEL
          bkgsub:
            title: Background level subtracted from data?
            type: boolean
            fits_keyword: BKGSUB`

Of course that means you'll also then need to update the way you reference them in the skymatch_step.py code, to include the extra layer of meta hierarchy, i.e. "image.meta.skymatch.bkglevel" and "image.meta.skymatch.bkgsub".